util/processinfo.go

```diff
@@ -169,6 +170,12 @@ type SessionManager interface {
 	KillAllConnections()
 	UpdateTLSConfig(cfg *tls.Config)
 	ServerID() uint64
+	// Put the internal session pointer to the map in the SessionManager
+	StoreInternalSession(addr unsafe.Pointer)
```

Comment:
Why use `unsafe.Pointer`?

Reply:
If I use Session interface, it causes package cycle.